packages/main/src/Select.js

```diff
@@ -378,10 +370,12 @@ class Select extends UI5Element {
 		}
 	}
 
-	_syncSelection() {
+	async _syncSelection() {
```

Comment:
This would be a problem, because this method is called during onBeforeRendering. We cannot await there because onBeforeRendering is strictly synchronous. What will happen is that the rendering part will continue without onBeforeRendering has finished. 

Instead, you should move the whole async part to onAfterRendering.